### DISCUSSION TITLE
Feature request: Disable gyro-driven increase in shutter speeds

### DISCUSSION THREAD

**Discussion by andrerav** [gopro/labs#165] [Feature Requests]
With the awesome EXPS setting enabled I noticed that when I move my camera around, the GoPro will automatically increase the shutter speed as a reaction to gyro input. I think this is a brilliant feature in many use cases since it will help hypersmooth perform well, however there's a tradeoff since the increase in shutter speed in many cases will drive up the ISO.

For me, as a complete and utter pixel-peeping nerd, it would be very nice to be able to disable (or adjust the sensitivity of) the gyro-based input so that the shutter speed only reacts to lighting conditions and not movement, and thus will only increase the ISO when the shutter speed limit is reached.

I think this feature would go nicely hand-in-hand with the EXPN and EXPX settings, and can potentially give an excellent boost to image quality.

**Comment by dnewman-gpsw** [comment]
Interesting idea.

**Reply by andrerav** [reply]
Good to hear. The more I think about it, the more I want it, hehe :) I tried disabling hypersmooth to see if that would stop the gyro from affecting the shutter speed, but that did not seem to be the case. The idea was that I could just stabilize the footage in post instead. But alas.  So it seems you're my only hope @dnewman-gpsw 🙏
